bower.json

```diff
@@ -13,12 +13,12 @@
     "angular-touch": "1.5.11",
     "angular-route": "1.5.11",
     "angular-bootstrap": "0.14.3",
-    "angular-patternfly": "4.3.0",
+    "angular-patternfly": "4.5.4",
```

Comment:
Should we take 4.5.6 now that it's out?

Reply:
Yeah, we probably should to get the empty state h1 issue fix.